examples/tutorial.dx

```diff
@@ -435,77 +500,91 @@ withState 10 $ \ state.
   integer tables (of type `n => Float32`).
 
 :t tableMean
+> ((n:Type) ?-> (n => Float32) -> Float32)
 
 ' If we try to apply `tableMean` to other types for get errors. For example,
   `tableMean` does not work when applied to a table of *pairs* of floats.
 
 :t (for (i, j). (x.i.j, x.i.j))
+> ((Fin 3 & Fin 8) => (Float32 & Float32))
 
 tableMean (for (i, j). (x.i.j, x.i.j))
+> Type error:
+> Expected: Float32
+>   Actual: (Float32 & Float32)
+>
+> tableMean (for (i, j). (x.i.j, x.i.j))
+>                         ^^^^^^^^^^^^
 
 ' Intuitively, supporting this seems possible. We just need to be able to
   add and divide pair types. Let's look closer at the exact types of the
   addition and division operators.
 
 :t (+)
+> ((a:Type) ?-> (Add a) ?=> a -> a -> a)
 
 :t (/)
+> ((a:Type) ?-> (VSpace a) ?=> a -> Float32 -> a)
 
 ' These function types are a bit complex.
   `(+)` maps `a -> a -> a` with a constraint that `a` implements the `Add'`
   interface. Whereas `(/)` maps `a -> Float32 -> a` where `a` implements the
   `VSpace'` interface.
 
-' If we look in the Prelude, we can see that these interfaces are defined as:
+' If we look in the Prelude, we can see that these interfaces are defined as
+ (This will throw error because it mirrors the prelude, but we are just repeating it here.):
 
-interface Add' a
-  add' : a -> a -> a
-  sub' : a -> a -> a
-  zero' : a
+interface Add a
+  add : a -> a -> a
+  sub : a -> a -> a
+  zero : a
+> Error: variable already defined: Add
```

Comment:
Yeah, looks like those re-definitions fail. Can you put them in some quoted blocks, or give them different names?